lib/std/stropt.nim

```diff
@@ -0,0 +1,71 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2021 Nim Contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+const whitespaces = {' ', '\t', '\v', '\r', '\l', '\f'}
```

Comment:
this could be public, and strutils could import + re-export it to avoid duplicating it

but ok to defer to future work

Reply:
But strutils already has [whitespace thing](https://nim-lang.github.io/Nim/strutils.html#Whitespace).

Reply:
the idea is for `stropt` to be more low-level than `strutils` so strutils and other modules can `import stropt` ( not the other way around) and then benefit from its optimizations, eg strutils.strip can be re-implemented more efficiently using stropt procs (ditto with other procs)